app/src/androidTest/java/com/sdp/movemeet/Activity/ActivityNotNullDescriptionTest.java

```diff
@@ -54,13 +69,51 @@
             DUMMY_DATE
     );
 
-    @Rule
-    public ActivityScenarioRule<ActivityDescriptionActivity> testRule = new ActivityScenarioRule<>(new Intent(getApplicationContext(), ActivityDescriptionActivity.class).putExtra("activity", activity));
+    @Before
+    public void signIn() {
+        CountDownLatch latch = new CountDownLatch(1);
+
+        mAuth = FirebaseAuth.getInstance();
+
+        mAuth.signInWithEmailAndPassword("movemeet@gmail.com", "password").addOnCompleteListener(new OnCompleteListener<AuthResult>() {
+            @Override
+            public void onComplete(@NonNull Task<AuthResult> task) {
+                if (task.isSuccessful()) {
+                    sleep(500);
```

Comment:
why do you need sleep where ?

Reply:
I don't know if it's really necessary, maybe I can delete it !